docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md

```diff
@@ -0,0 +1,313 @@
+| UnzerCheckoutDataResponseMapperPlugin | Maps `RestCheckoutDataTransfer.unzerCredentials.unzerKeypair.publicKey` to `RestCheckoutDataResponseAttributesTransfer.unzerPublicKey` while `RestCheckoutDataTransfer.unzerCredentials` is specified. | None          | SprykerEco\Glue\UnzerRestApi\Plugin\CheckoutRestApi                |
+| UnzerNotificationResource             | Adds Unzer notification resource.                                                                                                                                                                      | None          | SprykerEco\Glue\UnzerRestApi\Plugin\GlueJsonApiConventionExtension |
+
+**src/Pyz/Glue/CheckoutRestApi/CheckoutRestApiDependencyProvider.php**
+
+```php
+namespace Pyz\Glue\CheckoutRestApi;
+
+use Spryker\Glue\CheckoutRestApi\CheckoutRestApiDependencyProvider as SprykerCheckoutRestApiDependencyProvider;
+use SprykerEco\Glue\UnzerRestApi\Plugin\CheckoutRestApi\UnzerCheckoutDataResponseMapperPlugin;
+
+class CheckoutRestApiDependencyProvider extends SprykerCheckoutRestApiDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Glue\CheckoutRestApiExtension\Dependency\Plugin\CheckoutDataResponseMapperPluginInterface>
+     */
+    protected function getCheckoutDataResponseMapperPlugins(): array
+    {
+        return [
+            new UnzerCheckoutDataResponseMapperPlugin(),
+        ];
+    }
+}
+```
+
+**src/Pyz/Glue/GlueStorefrontApiApplication/GlueStorefrontApiApplicationDependencyProvider.php**
+
+```php
+namespace Pyz\Glue\GlueStorefrontApiApplication;
+
+use Spryker\Glue\GlueStorefrontApiApplication\GlueStorefrontApiApplicationDependencyProvider as SprykerGlueStorefrontApiApplicationDependencyProvider;
+use SprykerEco\Glue\UnzerRestApi\Plugin\GlueJsonApiConventionExtension\UnzerNotificationResource;
+
+class GlueStorefrontApiApplicationDependencyProvider extends SprykerGlueStorefrontApiApplicationDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Glue\GlueApplicationExtension\Dependency\Plugin\ResourceInterface>
+     */
+    protected function getResourcePlugins(): array
+    {
+        return [
+            new UnzerNotificationResource(),
+        ];
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+1. Create a cart by sending the `POST https://glue.mysprykershop.com/carts`.
+2. Add at least one item to the cart by sending `POST https://glue.mysprykershop.com/{{cart_uuid}}/items`.
+3. Check result by sending the `POST https://glue.mysprykershop.com/checkout-data?include=payment-methods` request.
+
+**Request Example:**
+```json
+{
+  "data":
+  {
+    "type": "checkout-data",
+    "attributes":
+    {
+      "idCart": "{{cart_uuid}}",
+      "payments": [
+        {
+          "paymentMethodName": "Unzer Sofort",
+          "paymentProviderName": "Unzer"
+        }
+      ]
+    }
+  }
+}
+```
+
+{% info_block infoBox %}
```

Comment:
If the steps listed in this infoBox are mandatory, then replace `{% info_block infoBox %}` with  `{% info_block warningBox %}`